### DISCUSSION TITLE
Adding a prompt in editor.js to integrate Generative Ai capabilities.

### DISCUSSION THREAD

**Discussion by jasonsprouse** [jscad/OpenJSCAD.org#1276] [Ideas]
I've started a discussion in the forum and am linking that convo [here.](https://openjscad.nodebb.com/topic/410/is-the-cli-package-also-included-in-the-web-version?_=1691636955539)

**Comment by z3dev** [comment]
@jasonsprouse Thanks for moving the discussion over here.

I really like the idea, and possibly some changes to V3 can be made to assist in the integration. So, what kind of fun things are we talking about? Do we need to supply some training, examples, etc to the AI?
